doc/api/fs.md

```diff
@@ -94,6 +94,12 @@ Error: EISDIR: illegal operation on a directory, read
     <stack trace.>
 ```
 
+*Note:* On Windows Node follows the concept of per-drive working directory.
```

Comment:
Should be `Node.js` although it's probably enough to just say `Windows follows...` as Node.js is not doing anything different from the OS behavior.